com.termius.termius.appdata.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- © 2019 Termius Corporation.> -->
+<component>
+ <id>com.termius.termius</id>
+ <metadata_license>CC0-1.0</metadata_license>
+ <project_license>LicenseRef-proprietary</project_license>
+ <name>Termius</name>
+ <summary>Termius is the SSH client that works on Desktop and Mobile</summary>
+ <description>
+  <p>
+  Securely Sync Data Across All Your Devices
+  Termius lets you organize hosts into groups.
+  Groups allow you to share settings, though each host can have its own separate preferences.
+  This data, along with connection and command history, is securely synced across all your devices.
+  Termius uses end-to-end encryption to ensure your data remains safe and secure.
+  </p>
+ </description>
+ <screenshots>
+  <screenshot type="default">
+   <image>https://assets.website-files.com/5c7036349b5477bf13f828cf/5c7eb7590c28be6ee5c8afdb_termius-p-2000.jpeg</image>
+   <caption>Termius is the SSH client that works on Desktop and Mobile</caption>
+  </screenshot>
+ </screenshots>
+ <url type="homepage">https://termius.com/</url>
+ <update_contact>hello@termius.com</update_contact>
```

Comment:
That seems to be upstream update_contact. This package is not made by upstream, isn't it?

Reply:
what sorry im not english what is your question ?

Reply:
Are you the upstream? This should be an addess you answer to

Reply:
fixed